django/test/runner.py

```diff
@@ -617,6 +618,7 @@ def get_test_runner_kwargs(self):
             'failfast': self.failfast,
             'resultclass': self.get_resultclass(),
             'verbosity': self.verbosity,
+            'stream': self.stream,
         }
```

Comment:
It's not necessary to add this publicly available feature, that would need to be documented and tested. Simply subclass `DiscoverRunner` and override `get_test_runner_kwargs`.

Reply:
Ah, good plan. Willdo.

Reply:
Done, will push soon.